### ISSUE TITLE
After #23239 was merged into master, intermediate workspaces in focus were not being removed

### ISSUE THREAD

**Issue by SamJenkins1** [mantidproject/mantid#23685]
### Expected behavior
Upon completing the ISIS Powder Gem Focus, intermediate workspaces would be removed
### Actual behavior
Intermediate workspaces were not deleted, filling ADS with many unnesecary workspaces named `focused_ws-`

Re #23239

**Comment by SamJenkins1** [comment]
Fixed in #23691
